wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -3858,6 +3859,22 @@ function ( $row ) use ( $name_map ) {
 		);
 	}
 
+	/**
+	 * Checks if column should define the default.
+	 *
+	 * @param stdClass $column The table column
+	 * @param string $mysql_type The MySQL data type
+	 *
+	 * @return boolean If column should have a default definition.
+	 */
+	private function column_has_default( $column, $mysql_type ) {
+		if ( null !== $column->dflt_value && '' !== $column->dflt_value && ! in_array( strtolower( $mysql_type ), array( 'datetime', 'date', 'time', 'timestamp', 'year' ), true ) ) {
+			return true;
```

Comment:
I noticed that some CREATE_TABLE statements also include `expires` as datetime e.g.:

```
`expires` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
```
Although this one has a value and not an empty string, should we also include it into this array in case it might come up with an empty string? What do you think?